neuralmonkey/runners/perplexity_runner.py

```diff
@@ -5,7 +5,7 @@
 computes its perplexities given the decoder.
 """
 
-# tests: lint
+# tests: lint, mypy
```

Comment:
Taky jsem přemýšlel, jestli je nemám všechny smazat. Možná bych to spíš nechal na nějaký cleanup commit, až nebudou rozjeté PR typu `decoder_refactor`, kde to bude komplikovat merge, když smažu řádek skoro v každém souboru. Ale jestli chceš, můžu to udělat hned.